### DISCUSSION TITLE
No summary on github PR?

### DISCUSSION THREAD

**Discussion by AArnott** [Tyrrrz/GitHubActionsTestLogger#35] [Q&A] [answered]
I'm trying to adopt your test logger (thanks!). The annotations in the diff view appeared as expected. But the summary comment on the main PR page didn't show up.

https://github.com/AArnott/Nerdbank.MessagePack/pull/165?new_mergebox=true

Any idea why?

**Comment by Tyrrrz** [comment, marked as answer]
I see it here: https://github.com/AArnott/Nerdbank.MessagePack/actions/runs/12321313659?pr=165

Is there some other place you expected to see it?

![image](https://github.com/user-attachments/assets/b724b6fb-357c-4367-b105-22eab5837965)

**Reply by AArnott** [reply]
Oh! 
Nah, that's just my ignorance as to how GitHub Actions works. I was looking at the PR page rather than the Actions page. Thanks.
